google/cloud/forseti/notifier/notifier.py

```diff
@@ -194,5 +199,8 @@ def run(inventory_index_id, service_config=None):
             violations_as_dict,
             notifier_configs.get('violation').get('findings').get('gcs_path'))
 
-    LOGGER.info('Notification complete!')
+    log_message = 'Notification completed!'
+    progress_queue.put(log_message)
+    progress_queue.put(None)
```

Comment:
why do we put a `None` here?

Reply:
The None is used as an identifier to identify the end of the queue, once the loop reached None, it will exit the loop and end the generator.

Reply:
Got it.  Thanks!